boxoffice/templates/index.html

```diff
@@ -2,6 +2,7 @@
 
 {%- block layoutheaders %}
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.4.11/c3.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-daterangepicker/2.1.25/daterangepicker.min.css">
```

Comment:
Should this be in Baseframe? We should also get C3 in there now as we're using it regularly.